projects/core/build/build.tokens.ts

```diff
@@ -0,0 +1,231 @@
+/*
+ * Copyright (c) 2016-2024 Broadcom. All Rights Reserved.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import * as csso from 'csso';
+import * as fs from 'fs';
+import * as ts from 'typescript';
+
+import { Token } from './token-utils';
+import { baseTheme } from './tokens';
+
+const tokens: Token[] = Object.entries(flattenTokens(baseTheme)).map(token => token[1]);
+const experimental = `This token format is currently experimental and may change in the future`;
+
+fs.mkdirSync('./dist/@cds/core/styles/', { recursive: true });
+fs.mkdirSync('./dist/@cds/core/tokens/', { recursive: true });
+fs.mkdirSync('./dist/@cds/core/styles/tokens/generated', { recursive: true });
+
+// Public API Tokens
+buildCSSTokens('./dist/@cds/core/styles/module.tokens.css');
+buildJSONTokens('./dist/@cds/core/tokens/tokens.json');
+buildJSTokens('./dist/@cds/core/tokens/tokens.ts');
+buildAndroidXMLTokens('./dist/@cds/core/tokens/tokens.android.xml');
+buildIOSSwiftTokens('./dist/@cds/core/tokens/tokens.ios.swift');
```

Comment:
I think these should be removed. The `./dist/@cds/core` folder is not published as a package, so this is dead code.

Reply:
Good catch, I removed some, but these I've missed.

Reply:
Done